catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -94,15 +94,7 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ExportCommand.class);
 
-  private static final SimpleDateFormat ISO_8601_DATE_FORMAT;
-
-  static {
-    ISO_8601_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH-mm-ss.SSS'Z'");
-    ISO_8601_DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
-  }
-
-  private static final Supplier<String> FILE_NAMER =
-      () -> "export-" + ISO_8601_DATE_FORMAT.format(Date.from(Instant.now())) + ".zip";
+  private static final Supplier<String> FILE_NAMER;
```

Comment:
It looks like it is being used in writing a file:
`FILE_NAMER = () -> "export-" + ISO_8601_DATE_FORMAT.format(Date.from(Instant.now())) + ".zip"`
So I believe it is named properly.

Reply:
Actually, since the Default formatting for an Instant is ISO 8601 -- we can get rid of the simple date format all together and just use 
```java
FILE_NAMER = () -> "export-" + Instant.now().toString();
```

(or sans the toString if you wish to juse have it implicitly called.)

Reply:
We could also stick it in a String format if so desired.
```java
() -> String.format("export-%s.zip", Instant.now())
```